implementations/elixir/ockam/ockam/lib/ockam/api/request.ex

```diff
@@ -38,7 +38,7 @@ defmodule Ockam.API.Request do
       end
 
     base =
-      Header.encode(%Header{
```

Comment:
Let's rename `Request.encode/1` -> `Request.encode!/1` for consistency. We can address mixed use of `:bare` and `CBOR` later.